app/services/push_service.rb

```diff
@@ -1,13 +1,17 @@
 class PushService
   def send_update_store(channel, app_name, data)
-    channel = Channel.add_app_name_suffix(channel, ADMIN_APP) if app_name == ADMIN_APP
+    if [ADMIN_APP, BROWSE_APP, STOCK_APP].include?(app_name)
```

Comment:
Check the code in Channel.rb but I don't think you need the `if` statement here. It's already taken care of.

Reply:
Yeah, I've made the changes. Thanks @steveyken 
here's the commit link https://github.com/crossroads/api.goodcity/pull/647/commits/868e249bd2344523814c6429b7e46d6dc1714da8